sdk/transaction-builder/src/error_explain.rs

```diff
@@ -36,6 +36,7 @@ mod tests {
     use move_core_types::{account_address::AccountAddress, ident_str};
 
     #[test]
+    #[ignore] // re-enable after cleaning up hardcoded code
```

Comment:
this should work?

Reply:
It fails here https://github.com/aptos-labs/aptos-core/blob/main/sdk/transaction-builder/src/error_explain.rs#L22, when loading a compile error map.  This error map is not updated when releasing the framework